wp-content/themes/pub/wporg-learn-2020/sidebar-lesson-plan.php

```diff
@@ -12,7 +12,7 @@
 ?>
 <aside class="lp-sidebar">
 	<div class="lp-details">
-		<ul>
+		<ul class="taxonomies-meta">
```

Comment:
I think this should follow the `lp` prefix convention
```suggestion
		<ul class="lp-details-meta">
```